tutor/specs/components/new-course/build-course.spec.cjsx

```diff
@@ -12,13 +13,13 @@ describe 'CreateCourse: saving new course', ->
     NewCourseActions.save.restore()
 
   it 'it transitions to ready after save', ->
-    wrapper = shallow(<BuildCourse />)
+    wrapper = mount(<BuildCourse />)
     expect(NewCourseActions.save.calledOnce).to.be.true
     expect(wrapper.text()).to.include('building')
-    expect(wrapper.find('Icon[type="refresh"]')).to.have.length(1)
-    NewCourseActions.created({id: 1})
+    expect(wrapper.find('.ox-loader')).to.have.length(1)
+    NewCourseActions.created({id: '1'})
     wrapper.setState({})
-    expect(wrapper.find('Icon[type="refresh"]')).to.have.length(0)
+    expect(wrapper.find('.ox-loader')).to.have.length(0)
     expect(wrapper.text()).to.include('course is ready')
     undefined
```

Comment:
No need to require enzyme, it's `mount` and `shallow` is exported globally in config/test/jest.setup.coffee